### PR TITLE
Added support for libraries and cli tools as well as debs packed with lzma tar balls 

### PR DESCRIPTION
**Pull request by CreatureSurvive**
- lzma tar support
- searches and installs dylibs in bootstrap/usr/lib
- searches and installs cli tools in bootstrap/usr/bin
- messages for when no tweaks or libraries were found
- completion message

### CODE REVIEW

**Comment by alon-z** [comment]
Testing on Fedora, no need for --z=lzma with the tar command. It just works.
Also, if I do use it, then if it's not lzma then it does'nt work (returns type of file not recognized).

Are you on macOS? Maybe things are different between the platforms.

**Comment by CreatureSurvive** [comment]
@alonikomax Yes I'm on macOS and the script fails on any package that is bundled with lzma, perhaps checking the tar file extension or the platform would resolve the issue.

File: install_script.sh [hunk 1]
```diff
@@ -3,30 +3,60 @@ mkdir  -p $1.dir
 OLD_PWD=$pwd
 
 # Extract
-#tar -xvf $1 -C $1.dir/
+#tar --lzma -xvf $1 -C $1.dir/
 cd $1.dir
 ar x ../$1
-tar -xvf data.tar.*
+tar --lzma -xvf data.tar.*
```
Comment (alon-z) [inline install_script.sh]: Change it to:
if [[ $(ls | grep data.tar.lzma ) ]]; then
tar --lzma -xvf data.tar.lzma
else
tar -xvf data.tar.*
fi

I think there will never be more than one data.tar.* file.

**Comment by alon-z** [comment]
@CreatureSurvive I added a comment on your commit with an if for the solution,

**Comment by CreatureSurvive** [comment]
@alonikomax perfect, I just committed the changes. it should be good now.

**Pull request merged**